featurebyte/service/online_serving.py

```diff
@@ -190,21 +208,18 @@ async def get_online_features_by_feast(
                 feast_store=feast_store,
             )
         else:
-            added_column_names = None
+            added_column_names = []
 
-        # Include point in time column if it is required
-        is_point_in_time_column_required = self._require_point_in_time_request_column(
-            feature_cluster
+        request_data_metadata = await self._process_request_columns(
+            request_data=request_data,
+            feature_ids=feature_list.feature_ids,
+            feature_cluster=feature_list.feature_clusters[0],
```

Comment:
Would it be better to retrieve the `feature_cluster` by finding those cluster which matches the feature store ID?

Reply:
Hmm I think that might be a bit of an overkill for now since we have the assumption that there is only a single feature cluster in a feature list across the codebase.